index.html

```diff
@@ -8,21 +8,30 @@
     <link rel="stylesheet" href="main.css">
 </head>
 <body>
-    <div class="wrapper">
             <header>
                 <nav class="nav-bar">
```

Comment:
- [ ] [optional] Kindly check on your mobile design it doesn't match with the design given;

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/173593884-22345b04-1064-4a59-818a-0941e043a674.png) |  ![image](https://user-images.githubusercontent.com/61408860/173593952-ac6fe825-307f-42f9-9ddf-8351ae993257.png)